### PR TITLE
Backup-DbaDatabaseCertificate - Checklist Changes

### PR DESCRIPTION
**Pull request by jpomfret**
Changes proposed in this pull request:
 - formatted help section
 - added examples
 - case changes

How to test this code: 
- [ ] 
- [ ] 

Has been tested on minimum requirements:
- [ ]  Powershell 3
- [ ]  Windows 7
- [ ]  SQL Server 2000

Has been tested on maximum requirements:
- [ ]  SQL Server vNext
- [ ]  Windows 10
- [ ]  Azure Database

Tests for tester:
- [ ] Working/useful help content, including link to command on dbatools web site
- [ ] All examples work as advertised
- [ ] Does not contain template content
- [ ] Does not contain excessive/unnecessary amounts of comments
- [ ] Works remotely
- [ ] Works locally
- [ ] Works on lower versions or throws error specifying version not supported
- [ ] Works with named instances
- [ ] Works with clustered instances
- [ ] Handles offline/read only databases
- [ ] Supports multiple servers (at the command line or piped from Get-SqlRegisteredServerName)
- [ ] No un-handled errors which stop the command working with multiple servers

### CODE REVIEW

**Comment by jpomfret** [comment]
The following two examples aren't working as I'd expect, can someone check these and let me know? I can work on a fix if they are indeed not working.

```
Backup-DbaDatabaseCertificate -SqlInstance Server1
Backup-DbaDatabaseCertificate -SqlInstance Server1 -Certificate Certificate1
```

File: functions/Backup-DbaDatabaseCertificate.ps1 [hunk 1]
```diff
@@ -179,24 +202,24 @@ Exports all certificates named CertTDE on the specified SQL Server, not specifyi
...
+					Stop-Function -Message "$certName from $db on $instance cannot be exported." -Continue -Target $cert -InnerErrorRecord $_
 				}
 			}
 		}
 	}
 	
 	process {
 		if (Test-FunctionInterrupt) { return }
-		foreach ($instance in $sqlinstance) {
+		foreach ($instance in $sqlInstance) {
```
Comment (wsmelton) [inline functions/Backup-DbaDatabaseCertificate.ps1]: This one should actually be `$SqlInstance` because it is a parameter for the function.

Comment (jpomfret) [inline functions/Backup-DbaDatabaseCertificate.ps1]: So close :) should be fixed in latest commit

**Comment by gbargsley** [comment]
What needs to be tested on this PR?

@jpomfret @wsmelton

**Comment by jpomfret** [comment]
@gbargsley there is no code/functionality changes in here, just formatting changes to go along with @wsmelton 1.0 spreadsheet. Help was reformatted, added examples, some changed capitalization.

**Comment by potatoqualitee** [comment]
I know you're waiting for me @jpomfret - I'll be able to respond in a couple hours or less.

**Comment by potatoqualitee** [comment]
will make a couple mods

**Comment by jpomfret** [comment]
Thanks. Let me know what you change so I don't do it again in the next commands haha :)

**Comment by potatoqualitee** [comment]
so far, i just removed some periods from the output messages (dont know what i was thinking!) and I'm gonna figure out why it skips master. I may also change the behavior of the password stuff.

**Pull request merged**